executing/executing.py

```diff
@@ -608,31 +582,37 @@ def match(a, b):
             index = lasti
 
             def opname(i):
-                return bc_list[i // 2].opname
+                return self.bc_list[i // 2].opname
```

Comment:
Please extract `self.bc_list[i // 2]` into a method and also use it for `positions` in find_node.